### ISSUE TITLE
Using the same seed kwarg returns different values between GlorotUniform and GlorotUniformV2

### ISSUE THREAD

**Issue by fernandobperezm** [keras-team/tf-keras#71]
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04.5 LTS
- TensorFlow installed from (source or binary): Pip (binary)
- TensorFlow version (use command below): v2.6.0-0-g919f693420e 2.6.0
- Python version: 3.7.12
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: NA
- GPU model and memory: NA
- Exact command to reproduce: [Colab notebook](https://colab.research.google.com/drive/13V8v7a7fCQQMroaLz1PGvgrQ5Wf81IRp?usp=sharing)

**Describe the problem**.
The random values got by GlorotUniform are different between the V1 and V2 APIs using the same method seed, as a consequence, it is not possible to get the same exact results obtained by the V1 version in V2.

**Describe the current behavior**.
Setting the `operation` seed returns different tensors when using GlorotUniform when imported from `tf.compat.v1` and `tensorflow.keras.initializers`

**Describe the expected behavior**.
Both tensors must be equal

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**.

- Do you want to contribute a PR? (yes/no): No
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing): NA

**Standalone code to reproduce the issue**.
[Colab link](https://colab.research.google.com/drive/13V8v7a7fCQQMroaLz1PGvgrQ5Wf81IRp?usp=sharing)

**Source code / logs**.
NA

P.S: Submitting bug here in the Keras repo due to comments in the TF repo, specifically: [comment in issue 52294 in TF](https://github.com/tensorflow/tensorflow/issues/52294#issuecomment-938548708)
